codegenerator/cli/src/config_parsing/entity_parsing.rs

```diff
@@ -603,10 +609,49 @@ impl Field {
         // TODO: should we dis-allow indexed fields that are either `id` or derived From fields?
         let is_indexed = indexed_count > 0;
 
-        let field_type =
+        let mut field_type =
             FieldType::from_obj_field_type(&field.field_type, derived_from_field, is_indexed)
                 .context(format!("Failed parsing field {}", field.name))?;
 
+        if numeric_directives.len() == 1 {
+            if let FieldType::RegularField {
+                field_type: UserDefinedFieldType::NonNullType(inner),
```

Comment:
Why do we only care about non null types?

Reply:
We were - chatgpt definitely removed it - lol